reader/components/highlighter/Highlighter.jsx

```diff
@@ -144,7 +148,7 @@ class Highlighter extends React.Component {
       <>
         {contextRoot && contextMenu}
         <Highlights updateContextMenu={this.onUpdateContextMenu} />
-        {children}
+        {child}
```

Comment:
Weird change for this PR

Reply:
I couldn't use children because of shadow variable